runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollConfig.java

```diff
@@ -36,10 +37,27 @@ public interface ClueScrollConfig extends Config
 	@ConfigItem(
 		keyName = "displayHintArrows",
 		name = "Display hint arrows",
-		description = "Configures whether or not to display hint arrows for clues."
+		description = "Configures whether or not to display hint arrows for clues.",
+		position = 0
 	)
 	default boolean displayHintArrows()
 	{
 		return true;
 	}
+
+	@ConfigSection(name = "Advanced", description = "Advanced options", position = 1, closedByDefault = true)
+	String advancedSection = "Advanced";
+
+	@ConfigItem(
+			keyName = "autoIdentifyClues",
+			name = "Auto-identify clue scrolls",
+			description = "Automatically identify clue scrolls when added to inventory",
+			warning = "Does not work for Beginner and Master clues due to their shared Item IDs",
```

Comment:
I don't know that this limitation is significant enough to merit bringing up a pop-up window to warn users of it.

Reply:
This was mostly an attempt to reduce support traffic for users which may become confused when Beginner and Master clue are not "Auto-identified"